test/test_aa.py

```diff
@@ -390,3 +462,43 @@ def testFindByNameCaseIgnoredNameWithSpace(self):
                 'volume': -0.389221556886,
             },
             aa.propertyDetails)
+        self.assertEqual(
+            {
+                'aliphaticity': 1,
+                'aromaticity': 1,
+                'composition': 2,
+                'hydrogenation': 1,
+                'hydropathy': 1,
+                'hydroxyethilation': 2,
+                'iep': 1,
+                'polar_req': 4,
+                'polarity': 4,
+                'volume': 3,
+            },
+            aa.propertyClusters)
+
+
+class TestPropertyClusters(TestCase):
+    """
+    Tests for the PROPERTY_CLUSTERS dict.
+    """
+    def testPropertyClusterKeys(self):
+        """
+        The PROPERTY_CLUSTERS dict must contain the right keys.
+        """
+        self.assertEqual(AA_LETTERS, sorted(PROPERTY_CLUSTERS.keys()))
+
+    def testNumberOfValues(self):
+        """
+        Each key in PROPERTY_CLUSTERS must have a dict with 10 elements in it
+        as the value.
+        """
+        for aa, propertiesDict in PROPERTY_CLUSTERS.items():
```

Comment:
Here I think you want `PROPERTY_CLUSTERS.values()` seeing as you don't use `aa` in the loop.